halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/NodeFilter.java

```diff
@@ -64,6 +64,8 @@ public NodeFilter anyMaster() {
     return this;
   }
 
+  public NodeFilter() { }
```

Comment:
Filter nothing, I think it'll be clearer if I wrap this into a static method called `makeRejectAllFilter()` or something similar.